src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java

```diff
@@ -5,10 +5,22 @@
  */
 package edu.iis.powp.command;
 
+import edu.iis.client.plottermagic.IPlotter;
 /**
  *
  * @author karko
  */
-public class CommandDrawLineToPosition {
+public class CommandDrawLineToPosition implements IPlotterCommand{
+
+    private int i, i1;
```

Comment:
Unmeaningful names of attributes

Reply:
Just using already existing names! Better to be consistent with the available code.